examples/src/bin/tlsserver-mio.rs

```diff
@@ -640,14 +642,29 @@ fn make_config(args: &Args) -> Arc<rustls::ServerConfig> {
 
     config.key_log = Arc::new(rustls::KeyLogFile::new());
 
-    if args.flag_resumption {
+    if args.flag_no_resumption {
+        config.session_storage = Arc::new(rustls::server::NoServerSessionStorage {});
+    } else {
         config.session_storage = rustls::server::ServerSessionMemoryCache::new(256);
     }
 
     if args.flag_tickets {
         config.ticketer = provider::Ticketer::new().unwrap();
     }
 
+    if args.flag_max_early_data > 0 {
+        if !&versions.contains(&&rustls::version::TLS13) {
```

Comment:
small nit: I think the `&` on `versions` is unnecessary here: 

```suggestion
        if !versions.contains(&&rustls::version::TLS13) {
```

Reply:
I have fixed it with https://github.com/rustls/rustls/pull/2059/commits/356822ba424cb91e70ba354230e3a1eaedea1cbc.